### PR TITLE
[WIN32K] Remove some instances of the ugly RETURN() macro

### PR DESCRIPTION
**Pull request by tkreuzer**

### CODE REVIEW

File: win32ss/user/ntuser/simplecall.c [hunk 1]
```diff
@@ -137,232 +142,281 @@ NtUserCallNoParam(DWORD Routine)
...
+        {
+            DWORD_PTR Result;
 
             if (!(Result = (DWORD_PTR)IntCreateCurIconHandle((DWORD)Param)))
             {
-               EngSetLastError(ERROR_NOT_ENOUGH_MEMORY);
-               RETURN(0);
+                EngSetLastError(ERROR_NOT_ENOUGH_MEMORY);
+                Result = 0;
+                break;
```
Comment (ThFabba) [inline win32ss/user/ntuser/simplecall.c]: Arguably this break is unnecessary

File: win32ss/user/ntuser/simplecall.c [hunk 2]
```diff
@@ -137,232 +142,281 @@ NtUserCallNoParam(DWORD Routine)
...
             }
+            _SEH2_END;
+            break;
+        }
+
+        case ONEPARAM_ROUTINE_REPLYMESSAGE:
+            Result = co_MsqReplyMessage((LRESULT)Param);
+            break;
+
+        default:
```
Comment (ThFabba) [inline win32ss/user/ntuser/simplecall.c]: Any reason this isn't at the bottom?

File: win32ss/user/ntuser/simplecall.c [hunk 3]
```diff
@@ -372,99 +426,120 @@ NtUserCallOneParam(
...
+    PWND Window;
+    DWORD_PTR Ret;
+    TRACE("Enter NtUserCallTwoParam\n");
+    UserEnterExclusive();
+
+    switch (Routine)
+    {
+        case TWOPARAM_ROUTINE_REDRAWTITLE:
+        {
             DWORD_PTR Ret;
```
Comment (ThFabba) [inline win32ss/user/ntuser/simplecall.c]: Shadowing variable. Surprised it doesn't give you an uninitialized warning for the outer one.

File: win32ss/user/ntuser/simplecall.c [hunk 4]
```diff
@@ -372,99 +426,120 @@ NtUserCallOneParam(
...
+        case TWOPARAM_ROUTINE_UNHOOKWINDOWSHOOK:
+            Ret = IntUnhookWindowsHook((int)Param1, (HOOKPROC)Param2);
+            break;
 
-CLEANUP:
-   TRACE("Leave NtUserCallTwoParam, ret=%p\n", (PVOID)_ret_);
-   UserLeave();
-   END_CLEANUP;
+        default:
+        ERR("Calling invalid routine number 0x%x in NtUserCallTwoParam(), Param1=0x%x Parm2=0x%x\n",
```
Comment (ThFabba) [inline win32ss/user/ntuser/simplecall.c]: Missing indent, also `Ret` is uninitialized in the default case.

File: win32ss/user/ntuser/simplecall.c (hunk 1 above)
Comment (tkreuzer) [reply win32ss/user/ntuser/simplecall.c]: Fixed.

File: win32ss/user/ntuser/simplecall.c (hunk 2 above)
Comment (tkreuzer) [reply win32ss/user/ntuser/simplecall.c]: No, I fixed it.

File: win32ss/user/ntuser/simplecall.c (hunk 3 above)
Comment (tkreuzer) [reply win32ss/user/ntuser/simplecall.c]: Fixed

File: win32ss/user/ntuser/simplecall.c (hunk 4 above)
Comment (tkreuzer) [reply win32ss/user/ntuser/simplecall.c]: Fixed

**Pull request merged**